Practice2/src/styles/variables/main-var.css

```diff
@@ -0,0 +1,33 @@
+:root {
+  /* Color */
+  --c-light: #fff;
+  --c-dark: #4b4b4b;
+  --c-warning: #EBCB80;
+
+  /* Color heading */
+  --c-heading-light: var(--c-light);
+  --c-heading-dark: var(--c-dark);
+
+  /* Color text */
+  --c-text-light: var(--c-light);
+  --c-text-dark: var(--c-dark);
+
+  /* Font size */
+  --fs-default: 16px;
+  --fs-medium: 18px;
+  --fs-large: 44px;
+  --fs-largest: 64px;
```

Comment:
What happen if later on the design changes and there is a font size of 70px added?

Reply:
I understand. I fixed the name of variable

Reply:
It's still not good enough with `wide` and `wider`, but just go ahead. We will talk about this in the review session.